app/Models/CountryStatistics.php

```diff
@@ -34,18 +34,37 @@ public static function recalculate($countryAcronym, $modeInt)
             ->where('rank_score', '>', 0)
             ->whereHas('user', function ($userQuery) {
                 return $userQuery->default();
-            })->select(DB::raw('sum(ranked_score) AS ranked_score, sum(playcount) AS playcount, count(*) AS usercount, sum(rank_score) AS rank_score'))
+            })->select(DB::raw('sum(ranked_score) AS ranked_score, sum(playcount) AS playcount, count(*) AS usercount'))
             ->first();
 
         if ($stats->ranked_score > 0) {
+            $userPerformances = UserStatistics\Model::getClass(Beatmap::modeStr($modeInt))
+                ::select('rank_score')
+                ->where('country_acronym', $countryAcronym)
+                ->where('rank_score', '>', 0)
+                ->whereHas('user', function ($userQuery) {
+                    return $userQuery->default();
+                })
```

Comment:
might as well make a query variable first and use it for both queries. Also the anonymous function can be shortened with `fn ($userQuery) => $userQuery->default()`